mesheryctl/doc/cmds.yml

```diff
@@ -0,0 +1,192 @@
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
+example: ""
+---
+layout: default
+title: cmds.yml
+permalink: reference/cmds.yml/main
+redirect_from: reference/cmds.yml/main/
+type: reference
+display-title: "false"
+language: en
+command: cmds.yml
+subcommand: nil
+---
+
+name: test
+description: ""
+usage: test [flags]
```

Comment:
I would like to know why this file was created, it seems you added the test keyword to be detected for the test coverage ?